### PR TITLE
Fixes a bunch of stuff in .github

### PR DESCRIPTION
**Pull request by Altoids1**
I redid all the templates for PRs, issues, and the like. As well, I changed the automatic-review-request stuff to include us, with Parune getting pinged on any change to any ``/icons/`` folder, Morrowwolf being pinged for ``/maps/`` changes, and both Morrow & I being pinged whenever something really important is touched.

### CONTRIBUTING.md
This thing is supposed to help newbies with getting started and having good-quality code, but this file is very confusing, full of Yog references, long, and poorly written. So I went through and rewrote a lot of the stuff in this file. Some of the extraneous, advanced tips on coding for DM were moved to [the Wiki](https://github.com/morrowwolf/Barony13/wiki/Advanced-Tips-on-BYOND-Programming).

You can read the new version I've written [here](https://github.com/morrowwolf/Barony13/blob/altoids-github/.github/CONTRIBUTING.md), since the diffs are large and really hard to parse by themselves.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -239,149 +280,22 @@ This prevents nesting levels from getting deeper then they need to be.
...
-
-* If you are proposing multiple changes, which change many different aspects of the code, you are expected to section them off into different pull requests in order to make it easier to review them and to deny/accept the changes that are deemed acceptable.
-
-* If your pull request is accepted, the code you add no longer belongs exclusively to you but to everyone; everyone is free to work on it, but you are also free to support or object to any changes being made, which will likely hold more weight, as you're the one who added the feature. It is a shame this has to be explicitly said, but there have been cases where this would've saved some trouble.
-
-* Please explain why you are submitting the pull request, and how you think your change will be beneficial to the game. Failure to do so will be grounds for rejecting the PR.
-
+* Put spaces in-between your math, logic, & bitwise operators. Our host is colourblind and ergo is not reading your code with a decent mark-up. Please be kind to him.
```
Comment (ThatLing) [inline .github/CONTRIBUTING.md]: rude

Comment (morrowwolf) [reply .github/CONTRIBUTING.md]: WOW
